app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -872,16 +952,16 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
         b.settingsActivityOnDeviceLastUpdatedTimeTxt.visibility = View.VISIBLE
         val timeStamp = persistentState.localBlockListDownloadTime
         b.settingsActivityOnDeviceLastUpdatedTimeTxt.text = getString(R.string.settings_local_blocklist_version, Utilities.convertLongToDate(timeStamp))
-        val path: String = requireContext().filesDir.canonicalPath + "/" +timeStamp
+        val path: String = requireContext().filesDir.canonicalPath + "/" + timeStamp
         if (appMode?.getBraveDNS() == null) {
```

Comment:
turn this into a guard statement, move it to the top of the method, return

Reply:
Made as guard statement.